docs/src/index.md

```diff
@@ -7,6 +7,13 @@ emd
 emd2
 ```
 
+### Exact optimal transport for 1D
+
+```@docs
+optimal_transport_cost
+optimal_transport_plan
```

Comment:
Hi @devmotion, 
Yes I agree that we can move away from mirroring the POT function naming scheme as currently. Although with some thought, I think earth mover's distance/EMD is clearer (at least for me) in that it refers specifically to the unregularised OT problem. So perhaps `emdcost` or `emdplan` would work? What do you think.

Reply:
My dissertation has been a literature review on Optimal Transport for Machine Learning. In this kind of literature, the term Earth-Movers Distance is not very usual (it's mostly used in the earlier papers), and it usually refers to the case where the cost is |x-y|, hence the name. So for me, I find it really confusing, because when I use the `emd` function, I tend to think that I'm only dealing with the distance as cost.

I find `otcost` and `otplan` to be the most direct and concise. I guess @devmotion  will be the decisive vote.

Reply:
I think I tend to `otcost` and `otplan` since it seems reasonably descriptive and still concise. For me, the terms optimal transport cost and plan already imply that we operate in the unregularized setting (similar to other areas in e.g. ML and statistics where maximum likelihood estimation is unregularized by default and one only specifies explicitly if it is regularized). In my opinion, the term is also better than "earth mover's distance" since the latter seems to be a more recently introduced name in the field of computer vision (see e.g. the introduction in Computational Optimal Transport and the [Wikipedia page to Earth mover's distance](https://en.wikipedia.org/wiki/Earth_mover%27s_distance#History)).

That being said, we can always revisit later and rename the function if it turns out that it was a bad choice.

Reply:
Ok yes, I agree. I think we should separate the words with an underscore `ot_cost` and `ot_plan`, since `sinkhorncost` and `sinkhornplan` are a bit long.